packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 46.3.0 - June 27, 2019
+
+### Fixed
+
+- Update launch configurations for running extension tests ([PR #1422](https://github.com/forcedotcom/salesforcedx-vscode/pull/1422))
```

Comment:
This change is not part of any package so i don't have a package name to include for this change

Reply:
We don't need to include that change since it's exclusive to how we develop and has no customer impact.

Reply:
@lcampos removed that change